src/Anouar/Fpdf/Fpdf.php

```diff
@@ -980,32 +980,30 @@ function SetXY($x, $y)
 	$this->SetX($x);
 }
 
-function Output($name='', $dest='')
+function Output($dest='', $name='')
```

Comment:
@m-lopez-f what is wrong with the current params definition ?

Reply:
When i saw [doc](http://www.fpdf.org/es/doc/output.htm) about this project, this function indicate that the first parameter of function is dest and second name. I used this function with your definition and throw a error: FPDF error: Incorrect output destination: XXXX.PDF. 
But if changed order this problem has resolved.